src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.ts

```diff
@@ -14,34 +20,67 @@ import {
 })
 export class InlineFormDemoComponent {
 
-  public demoModel: {
-    firstName?: string;
-  } = { };
-
   public firstName: string = 'Jane';
 
+  public myForm: FormGroup;
+
   public inlineFormConfig: SkyInlineFormConfig = {
     buttonLayout: SkyInlineFormButtonLayout.Custom,
     buttons: [
-      { action: 'save', text: 'Save', styleType: 'primary' },
-      { action: 'delete', text: 'Delete', styleType: 'default' },
-      { action: 'reset', text: 'Reset', styleType: 'default' },
-      { action: 'cancel', text: 'Cancel', styleType: 'link' }
+      {
+        action: 'save',
+        text: 'Save',
+        styleType: 'primary'
+      },
+      {
+        action: 'delete',
+        text: 'Delete',
+        styleType: 'default'
+      },
+      {
+        action: 'reset',
+        text: 'Reset',
+        styleType: 'default'
+      },
+      {
+        action: 'cancel',
+        text: 'Cancel',
+        styleType: 'link'
+      }
     ]
   };
 
   public showForm: boolean = false;
 
-  constructor() {
-    this.demoModel.firstName = this.firstName;
+  constructor(private formBuilder: FormBuilder) {
+    this.myForm = this.formBuilder.group({
```

Comment:
I'd throw this code in a `ngOnInit` (at least) to demonstrate better practices.

Reply:
Done on all of them